src/fhir/guides/ig-atf/Home/Einfuehrung.page.md

```diff
@@ -1,41 +1,76 @@
 ---
-topic: Einfuehung
+topic: Einfuehrung
 ---
 
 # {{page-title}}
 
 ## Motivation und Hintergrund
 
-Im deutschen Gesundheitswesen etabliert sich FHIR als Standard für den Austausch medizinischer Informationen. Zusätzlich existiert bereits mit [KIM](https://www.gematik.de/anwendungen/kim) (Kommunikation im Gesundheitswesen) ein Kommunikationsprotokoll, um den sicheren Austausch von medizinischen Informationen zu gewährleisten.
+FHIR etabliert sich zunehmend als Standard für den Austausch medizinischer Informationen im deutschen Gesundheitswesen. Parallel dazu bietet [KIM](https://www.gematik.de/anwendungen/kim) (Kommunikation im Gesundheitswesen) bereits ein sicheres Kommunikationsprotokoll für den medizinischen Datenaustausch.
 
-Diese Spezifikation soll den Austausch von strukturierten Daten basierend auf FHIR für diverse Austauschplattformen und -protokolle ermöglichen.
+Mit dieser Spezifikation wird der strukturierte Datenaustausch auf Basis von FHIR weiterentwickelt und ermöglicht die flexible Nutzung über verschiedene Austauschplattformen und -protokolle wie KIM und TIM. Ziel ist es, eine einheitliche, interoperable Lösung für den sicheren, effizienten Datenaustausch zu schaffen.
 
-## Konzept
+## Wie funktioniert das ATF?
 
-Diese Spezifikation basiert auf dem "Messaging"-Konzept von FHIR. Dafür gibt es die Profile `BundleAppTransportFramework` (MessageBundle) und `MessageHeaderAppTransportFramework` (MessageBundle).
+Das App-Transport-Framework (ATF) ermöglicht eine strukturierte Kommunikation zwischen einem Sender und einem Empfänger, basierend auf standardisierten FHIR-Ressourcen. Die gesendete Nachricht enthält einen Betreff, der den Nachrichtentyp im jeweiligen Anwendungsfall festlegt, und einen Nachrichtentext, der bei Bedarf über XSLT generiert werden kann. Ein FHIR Bundle wird als Anhang hinzugefügt, das die eigentlichen Daten der Nachricht enthält.
```

Comment:
"der bei Bedarf über XSLT generiert werden kann" ist nicht korrekt. Zumindest nicht "bei Bedarf". Das ist etwas was vom Spezifikateur eines Anwendungsfalls erstellt werden muss.
Das würde ich auf der Ebene im ATF glaube ich gar nicht erwähnen. Das ist etwas, was wir in der Rezeptanforderung machen.

Man kann im ATF lediglich darauf hinweisen, dass sowas möglich ist und Spezifikateure von Anwendungsfällen dann auch das anbieten können.

Reply:
Geändert in 
> Das App-Transport-Framework (ATF) ermöglicht eine strukturierte Kommunikation zwischen einem Sender und einem Empfänger, basierend auf standardisierten FHIR-Ressourcen. Die gesendete Nachricht enthält einen Betreff, der den Nachrichtentyp im jeweiligen Anwendungsfall festlegt, und einen Nachrichtentext, der bei Bedarf durch Spezifikateure eines Anwendungsfalls mit Transformationsmethoden wie XSLT erstellt werden kann. Ein FHIR-Bundle wird als Anhang hinzugefügt, das die eigentlichen Daten der Nachricht enthält.

Reply:
top!